extensions/renderer/resources/app_window_custom_bindings.js

```diff
@@ -30,9 +30,10 @@ var try_hidden = function (view) {
 };
 
 var try_nw = function (view) {
-  if (view.nw)
-    return view;
-  return privates(view);
+  if (view.nw) {
+    return true;
+  }
+  return false;
 };
```

Comment:
But this would bring bugs on the app that not use nw.Window, which happens on #5528:
"Cannot read property 'Window' of undefined" "

This change should not block the case that expects objects, because the original "try_nw(win)" just return the parameter "win".